support/NseUpdaterManager.ts

```diff
@@ -1,42 +1,43 @@
-import * as fs from 'fs';
-import { BUNDLE_SHORT_VERSION_TEMPLATE_REGEX, BUNDLE_VERSION_TEMPLATE_REGEX, GROUP_IDENTIFIER_TEMPLATE_REGEX } from './iosConstants';
-import { OneSignalLog } from './OneSignalLog';
-import { ReaderManager } from './ReaderManager';
+import { FileManager } from './FileManager';
+import {
+  BUNDLE_SHORT_VERSION_TEMPLATE_REGEX,
+  BUNDLE_VERSION_TEMPLATE_REGEX,
+  GROUP_IDENTIFIER_TEMPLATE_REGEX,
+  NSE_TARGET_NAME
+} from './iosConstants';
+
+// project `ios/OneSignalNotificationServiceExtension` directory
+const entitlementsFileName =`OneSignalNotificationServiceExtension.entitlements`;
+const plistFileName = `OneSignalNotificationServiceExtension-Info.plist`;
 
-const entitlementsFilePath =`${__dirname}/serviceExtensionFiles/OneSignalNotificationServiceExtension.entitlements`;
-const plistFilePath = `${__dirname}/serviceExtensionFiles/OneSignalNotificationServiceExtension-Info.plist`;
-
-const logIfError = (err: NodeJS.ErrnoException | null) => {
-  if (err) {
-    OneSignalLog.error("Error updating OneSignal NSE Entitlement File.");
+export default class NseUpdaterManager {
+  private nsePath = '';
+  constructor(iosPath: string) {
+    this.nsePath = `${iosPath}/${NSE_TARGET_NAME}`;
   }
-}
 
-export default class NseUpdaterManager {
-  static async updateNSEEntitlements(groupIdentifier: string) {
-    let entitlementsFile = await ReaderManager.readFile(entitlementsFilePath);
+  async updateNSEEntitlements(groupIdentifier: string): Promise<void> {
+    const entitlementsFilePath = `${this.nsePath}/${entitlementsFileName}`;
+    let entitlementsFile = await FileManager.readFile(entitlementsFilePath);
     entitlementsFile = entitlementsFile.replace(GROUP_IDENTIFIER_TEMPLATE_REGEX, groupIdentifier);
-
-    fs.writeFile(entitlementsFilePath, entitlementsFile, 'utf-8', (err) => {
-      logIfError(err);
-    })
+    await FileManager.writeFile(entitlementsFilePath, entitlementsFile);
   }
 
-  static async updateNSEBundleVersion(version: string) {
-    let plistFile = await ReaderManager.readFile(plistFilePath);
+  async updateNSEBundleVersion(version: string): Promise<void> {
+    console.log("updateNSEBundleVersion:", version);
```

Comment:
Instead of doing `console.log` here shouldn't this call `OneSignalLog.log` instead?

There are more lines in this comment adding `console.log` below.

Reply:
for the record: it was an accident!